### PR TITLE
Fix github flows

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Reasoning behind the pull request
- some workflow builds were failing
  
## Proposed changes
- fix build flows, as shown in the following result screenshot
image

## Testing procedure
- N/A

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -48,16 +47,13 @@ jobs:
           GOPATH=$(go env GOPATH)
           ARCHIVE="elrond_""$APP_VER_SHORT""_""$GOOS""_""$GOARCH"".tgz"
           BUILD_DIR=${GITHUB_WORKSPACE}/build
-          ARWEN_VERSION=$(cat go.mod | grep arwen | sed 's/^.*arwen-wasm-vm *//' | tail -n 1)
-          ARWEN_PATH=${GITHUB_WORKSPACE}/build/arwen
-          WASMER_DIR=${GOPATH}/pkg/mod/github.com/\!elrond\!network/arwen-wasm-vm@${ARWEN_VERSION}/wasmer
+          WASM_VERSION=$(cat go.mod | grep wasm | sed 's/^.*wasm-vm-v1_4 *//' | tail -n 1)
+          WASMER_DIR=${GOPATH}/pkg/mod/github.com/\!elrond\!network/wasm-vm-v1_4@${WASM_VERSION}/wasmer
```
Comment (AdoAdoAdo) [inline .github/workflows/create_release.yml:51]: do we still need this?

Comment (iulianpascalau) [reply .github/workflows/create_release.yml:51]: yes, the libwasmer directory

**Review by SebastianMarian — APPROVED** [review]

File: .github/workflows/create_release.yml (hunk 1 above)
Comment (AdoAdoAdo) [reply .github/workflows/create_release.yml:51]: right, got confused was thinking about the arwen path

**Review by AdoAdoAdo — APPROVED** [review]

File: .github/workflows/create_release.yml (hunk 1 above)
Comment (iulianpascalau) [reply .github/workflows/create_release.yml:51]: This was the main reason the workflow failed. The following lines are fetched from this failed create release build flow:
https://github.com/ElrondNetwork/elrond-go/actions/runs/3655452536/jobs/6176794677

> ARWEN_PATH: /Users/runner/work/elrond-go/elrond-go/build/arwen
> WASMER_DIR: /Users/runner/go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@/wasmer
> cp: /Users/runner/go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@/wasmer/libwasmer_darwin_amd64.dylib: > No such file or directory
> Error: Process completed with exit code 1.

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**